### DISCUSSION TITLE
Wind Direction in Point-Stat

### DISCUSSION THREAD

**Discussion by CPKalb** [dtcenter/METplus#1738] [Configuration] [answered]
Hi:

I'm wondering about verifying wind direction using point_stat.  My model data is in grib2 format, with the wind variables UGRD and VGRD, so MET is deriving wind direction.  I have my METplus config set as follows:
FCST_POINT_STAT_VAR4_NAME = WDIR
FCST_POINT_STAT_VAR4_LEVELS = Z10

OBS_POINT_STAT_VAR4_NAME = wdir
OBS_POINT_STAT_VAR4_LEVELS = Z10

However, when I try to run this, I get the following message:
ERROR  :
ERROR  : PointStatVxOpt::process_config() -> wind direction may not be verified using point_stat.
ERROR  :

I've heard it's not possible to process wind direction directly in point stat, and that the data needs to be written as matched pairs and run through stat-analysis.  However, my model has data as the u and v wind components, and the observations are speed and direction.  Is there a fast way to match these up, or will the observations need to be converted to u and v?

Christina

**Comment by JohnHalleyGotway** [comment, marked as answer]
@CPKalb, this is true. Wind direction is NOT evaluated directly by the MET tools. And yes, I agree that the [Point-Stat](https://met.readthedocs.io/en/latest/Users_Guide/point-stat.html?highlight=wind%20vector#point-stat-tool) and/or other chapters of the user's guide should be updated to state this clearly. The reason why is that wind direction is not a typical scalar field that can be evaluated using continuous statistics... because it wraps at 360 degrees. To evaluate winds, user's are encouraged to verify the U and V components and request both VL1L2 and VCNT output line types. The VL1L2 line contains vector-based partials sums and the VCNT line type contains vector-continuos statistics.

The VCNT stats are described in [Appendix G](https://met.readthedocs.io/en/latest/Users_Guide/appendixG.html).

**Reply by CPKalb** [reply]
Thanks.  Is there a specific naming convention for the u and v components?  I have VL1L2 and VCNT turned on, but I'm not getting these output lines.  Although I am getting matched pair output for the u and v components.  I'm sure I'm missing a setting somewhere.

**Reply by CPKalb** [reply]
Actually, I think I figured it out.  It looks like if I set: is_u_wind = TRUE; and is_v_wind = TRUE; then I'm able to get VL1L2 and VCNT lines.
